lib/hybrid.h

```diff
@@ -68,19 +70,57 @@ struct HybridSimulator final {
   using Fuser = FuserT<IO, GateHybrid>;
   using GateFused = typename Fuser::GateFused;
 
+  /**
+   * Contextual data for hybrid simulation.
+   */
   struct HybridData {
+    /**
+     * List of gates on the "0" side of the cut.
+     */
     std::vector<GateHybrid> gates0;
+    /**
+     * List of gates on the "1" side of the cut.
+     */
     std::vector<GateHybrid> gates1;
+    /**
+     * List of gates on the cut.
+     */
     std::vector<GateX> gatexs;
+    /**
+     * Global qubit index to local qubit index map.
+     */
     std::vector<unsigned> qubit_map;
+    /**
+     * Number of qubits on the "0" side of the cut.
+     */
     unsigned num_qubits0;
+    /**
+     * Number of qubits on the "1" side of the cut.
+     */
     unsigned num_qubits1;
-    unsigned num_gatexs;  // Number of gates of the cut.
+    /**
+     * Number of gates on the cut.
+     */
+    unsigned num_gatexs;
   };
 
+  /**
+   * User-specified parameters for hybrid simulation.
+   */
   struct Parameter {
+    /**
+     * Fixed bitstring indicating values to assign to indices of prefix gates.
```

Comment:
It might be confusing in this form. `prefix` contains indices (one or two bits per index) of one-qubit gates in the Schmidt decompositions of prefix gates, i.e. `prefix` specifies the path for prefix gates.

Reply:
My understanding is that each prefix gate on the cut gets decomposed into a pair of one-qubit gates with one or two "extra" shared indices, and that `prefix` contains the values assigned to those indices. Is this incorrect? Would labeling the indices as "cut indices of prefix gates" be clearer?

Reply:
Yes, each prefix gate on the cut gets decomposed as a sum of pairs of one-qubit gates. The index we are talking about is a term index in this sum. So perhaps "Schmidt decomposition indices of prefix gates" will be clearer.